urdf/outdoornav_description.urdf.xacro

```diff
@@ -343,13 +350,31 @@
         <xacro:d435_gazebo name="rear_d435" topic_n="$(arg rear_d435_num)" />
       </xacro:if>
 
-      <!-- Axis Q62 PTZ camera -->
+      <!-- Axis PTZ camera -->
       <xacro:if value="$(arg axis_ptz_enable)">
-        <xacro:if value="$(arg axis_q62_enable)">
-          <xacro:axis_q62 prefix="ptz_camera" parent="base_link" axis_ptz_num="$(arg axis_ptz_num)">
-            <origin xyz="$(arg axis_q62_xyz)" rpy="$(arg axis_q62_rpy)" />
-          </xacro:axis_q62>
-        </xacro:if>
+        <link name="axis_ptz_camera_link"/>
```

Comment:
There's a URDF macro for the PTZ dome cameras: https://github.com/ros-drivers/axis_camera/blob/noetic-devel/axis_description/urdf/axis_dome_ptz.urdf.xacro. Is it worth using that here & below for the rear camera further down?